types/json/binary.go

```diff
@@ -148,6 +148,15 @@ var (
 	endian   = binary.LittleEndian
 )
 
+const (
```

Comment:
I think we can add a comment here. e.g.
```
// -------------------------------------------------------------------------------
// | header                        | key                                    | value
// -------------------------------------------------------------------------------
// | element size | data size | key off | key length | val type
// -------------------------------------------------------------------------------
// | 8 bytes length (little endian) |  x bytes | 1 byte
// -------------------------------------------------------------------------------
```

Reply:
@jackysp 
The full layout of binary json can be found at `serdes.go`, we can move it to `binary.go` when we delete the `serdes.go`.

Reply:
ok